docs/upgrade/index.rst

```diff
@@ -11,24 +11,30 @@ Upgrade ScyllaDB
    ScyllaDB Open Source to ScyllaDB Enterprise <upgrade-to-enterprise/index>
    ScyllaDB AMI <ami-upgrade>
 
+Overview
+---------
 
 ScyllaDB upgrade is a rolling procedure - it does not require a full cluster shutdown and is performed without any 
 downtime or disruption of service.
 
-To ensure a successful upgrade and avoid breaking anything, you should perform your upgrades consecutively - to each 
-successive X.Y version. For example, to upgrade from version 4.4 to 5.0, you should first upgrade ScyllaDB to version 
-4.5, next from 4.5 to 4.6, and finally, from 4.6 to 5.0, **without skipping any major version**.
+To ensure a successful upgrade, follow the :ref:`documented upgrade procedures <upgrade-upgrade-procedures>` tested by ScyllaDB. This means that:
 
-.. raw:: html
+* You should perform the upgrades consecutively - to each successive X.Y version, **without skipping any major version**.
```

Comment:
```suggestion
* You should perform the upgrades consecutively - to each successive X.Y version, **without skipping any minor version**.
```

Reply:
@avikivity - I vaguely remember this was not true (but perhaps untested?)

Reply:
I've pushed the commit to correct the information. The info from Avi on Slack: [ttps://scylladb-users.slack.com/archives/C2NLNBXLN/p1648172313885899](https://scylladb-users.slack.com/archives/C2NLNBXLN/p1648172313885899)